lua/gitlab/state.lua

```diff
@@ -60,14 +60,71 @@ M.settings = {
     insecure = true,
   },
   attachment_dir = "",
-  help = "g?",
-  popup = {
-    keymaps = {
+  keymaps = {
+    disable_all = false,
+    help = "g?",
+    global = {
+      disable_all = false,
+      add_assignee = "glaa",
+      delete_assignee = "glad",
+      add_label = "glla",
+      delete_label = "glld",
+      add_reviewer = "glra",
+      delete_reviewer = "glrd",
+      approve = "glA",
+      revoke = "glR",
+      merge = "glM",
+      create_mr = "glC",
+      choose_merge_request = "glc",
+      start_review = "glS",
+      summary = "gls",
+      copy_mr_url = "glu",
+      open_in_browser = "glo",
+      create_note = "gln",
+      pipeline = "glp",
+      toggle_discussions = "gld",
+      toggle_draft_mode = "glD",
+      publish_all_drafts = "glP",
+    },
+    popup = {
+      disable_all = false,
       next_field = "<Tab>",
       prev_field = "<S-Tab>",
+      perform_action = "ZZ",
+      perform_linewise_action = "<leader>l",
```

Comment:
If we are not using leaders universally this should match the other "action" keymaps, so probably ZL would work? I find it odd this is the only keymap prefixed with the leader key

Reply:
I've changed that to "ZA" because that's easier to type than "ZL" on qwerty or colemak (no change of hands). What do you think?

Reply:
There is still `<leader>p` for printing a discussion node, but that's for debugging, so I believe it can stay that way.